script/testing/util/test_server.py

```diff
@@ -160,22 +172,32 @@ def restart_db(self):
         self.stop_db()
         self.run_db()
 
+    def print_output(self, filename):
+        """ Print out contents of a file """
+        fd = open(filename)
+        lines = fd.readlines()
+        for line in lines:
+            LOG.info(line.strip())
+        fd.close()
+        return
+
     def run_test(self, test_case: TestCase):
         """ Run the tests """
         if not test_case.test_command or not test_case.test_command_cwd:
             msg = "test command should be provided"
-            raise RuntimeError(msg) 
+            raise RuntimeError(msg)
 
         # run the pre test tasks
         test_case.run_pre_test()
 
         # run the actual test
-        with open(test_case.test_output_file, "w+") as test_output_fd:
-            ret_val, _, _ = run_command(test_case.test_command,
-                                        test_case.test_error_msg,
-                                        stdout=test_output_fd,
-                                        stderr=test_output_fd,
-                                        cwd=test_case.test_command_cwd)
+        self.test_output_fd = open(test_case.test_output_file, "w+")
```

Comment:
Any reason to switch away from using the `with open`?

Reply:
fixed